tools.go

```diff
@@ -0,0 +1,8 @@
+//go:build tools
+// +build tools
```

Comment:
https://pkg.go.dev/cmd/go#hdr-Build_constraints

- `// +build` is an old syntax, used up to Go 1.16. Since it's already way past EOL, I think it makes sense to remove it
- As the doc states, convention is to use `//go:build ignore` for any files that shouldn't be built. This isn't special, just conventional (it's OK to use `tools`, it doesn't matter, I just think it helps documenting that this file isn't meant to be built)

Reply:
leaving `tools` as it looks like it is a convention for this file in particular.